src/ros_can_nodes/src/CANMsgRouter.cpp

```diff
@@ -31,7 +36,7 @@ int main(int argc, char **argv) {
 
 void CANMsgRouter::init() {
     // TODO: either fail on bad open_port OR have reconnect policy
-    int err = CANHelpers::open_port("can0");
+    int err = CANHelpers::open_port("vcan0");
```

Comment:
you need to change this back - maybe we should make it a ros param so its easy to change for testing

Reply:
👍

Reply:
gone for a argc/argv approach for now